Assets/Scripts/LoadingScreen.cs

```diff
@@ -97,14 +100,26 @@ public static void ResetCounter()
         loadingCounter = 0;
     }
 
-    public void Show()
+    public void Show(Camera transitionCamera)
     {
         if (enabled)
             return;
 
         enabled = true;
-        gameObject.transform.GetChild(0).gameObject.SetActive(true);
+        gameObject.transform.GetChild(1).gameObject.SetActive(true);
+        transitionScreen.worldCamera = transitionCamera;
```

Comment:
For readability we could have the transition to/from loading in separate methods, perhaps sharing a bunch of code?

Reply:
Refactored to one helper method 👍